teacher-webapp/src/callPage.js

```
@@ -1,52 +1,75 @@
-import React, { useState, useEffect } from 'react';
-import { useConference } from './context/ConferenceContext';
-import { startConferenceCall, endConferenceCall, sinkConferenceCall, muteParticipant, unmuteParticipant, playAudio, pauseAudio, addParticipant, resumeAudio } from './services/apiService';
-import { AddParticipantModal } from './components/AddParticipantModal';
-import { students as allStudents } from './state';
-import App from './App';
+import React, { useState, useEffect } from "react";
+import { useConference } from "./context/ConferenceContext";
+import {
+  startConferenceCall,
+  endConferenceCall,
+  sinkConferenceCall,
+  muteParticipant,
+  unmuteParticipant,
+  playAudio,
+  pauseAudio,
+  addParticipant,
+  resumeAudio,
+  seekAudio,
+} from "./services/apiService";
+import { AddParticipantModal } from "./components/AddParticipantModal";
+import { AudioContentModal } from "./components/AudioContentModal";
+import { SeekControls } from "./components/SeekControls";
+import { students as allStudents } from "./state";
+import App from "./App";
+
+const getPhoneNumber = (user) => user?.phoneNumber ?? user?.phone_number ?? "";
```

I don't think we can mute on unmute participants which have "" phone number. Ideally we are capturing phone details for all teachers and students , so maybe okay to error out if phone is not present.